src/sentry/models/artifactbundle.py

```diff
@@ -97,6 +111,29 @@ def delete_file_for_artifact_bundle(instance, **kwargs):
 post_delete.connect(delete_file_for_artifact_bundle, sender=ArtifactBundle)
 
 
+@region_silo_only_model
+class ArtifactBundleIndex(Model):
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    release_name = models.CharField(max_length=250)
+    # We use "" in place of NULL because the uniqueness constraint doesn't play well with nullable fields, since
+    # NULL != NULL.
```

Comment:
In the future Armin talks about where you can upload without a release, the `release_name` should also default to the empty string, right?